exports.getRecipes = async (req, res) => {
  try {

  } catch (err) {
    res.status(404).send(err.message);
  }
};

exports.getRecipe = async (req, res) => {
  try {

  } catch (err) {
    res.status(404).send(err.message);
  }
};
